### ISSUE TITLE
get error after excute `inshellisense bind`

### ISSUE THREAD

**Issue by HomyeeKing** [microsoft/inshellisense#31]
**Describe the bug**
![image](https://github.com/microsoft/inshellisense/assets/49113249/3a0d7269-6d5b-4267-862a-3e577abdd672)

**To Reproduce**

**Expected behavior**

**Environment**
 - OS: [e.g. iOS]  macOS
 - Output of `inshellisense --version`:  0.0.1-rc.2
 - Nodejs Version: [e.g. 20.9.0]  v16.19.0

**Additional context**

**Comment by User670** [comment]
Also having this issue on Windows. I already uninstalled this so I can't obtain a version number. A friend sent a screenshot of the error, then did `scoop shim add pwsh <path to powershell.exe>`, then `inshellisense bind` worked.

(is `pwsh` a thing for the command line? I get an error saying it's not a thing when I typed it, and it requires me to type `powershell` instead if I want powershell.)

**Comment by sktzofrenic** [comment]
I'm having the same issue on Ubuntu

```
➜  ~ inshellisense bind
Select your desired shell for keybinding creation
Loading...
node:internal/errors:478
    ErrorCaptureStackTrace(err);
    ^

Error: spawn powershell EACCES
    at Process.ChildProcess._handle.onexit (node:internal/child_process:285:19)
    at onErrorNT (node:internal/child_process:485:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  errno: -13,
  code: 'EACCES',
  syscall: 'spawn powershell',
  path: 'powershell',
  spawnargs: [ '-c', 'echo $profile' ],
  cmd: 'echo $profile',
  stdout: '',
  stderr: ''
}
```

**Comment by Achill113** [comment]
+1

**Comment by Nisthar** [comment]
also getting the same error

**Comment by cpendery** [comment]
This was fixed by #29 and will be included in the next pre-release later today. I'll close this once that release it out

**Comment by cpendery** [comment]
The new version is [released](https://www.npmjs.com/package/@microsoft/inshellisense) and I'm not seeing the issue anymore on Linux
